plugins/frontend/backstage-plugin-argo-cd/src/api/index.ts

```diff
@@ -93,8 +94,7 @@ export class ArgoCDApiClient implements ArgoCDApi {
     const result = Object.keys(params)
       .filter(key => params[key] !== undefined)
       .filter(
-        key =>
-          this.useNamespacedApps || key === ARGOCD_ANNOTATION_APP_NAMESPACE,
```

Comment:
`ARGOCD_ANNOTATION_APP_NAMESPACE` could never be a key here since we aren't passing an annotation, we're passing through the `appNamespace` query param in all cases.

I think the intent is to strip the `appNamespace` query param if the Backstage instance is not opted in, please correct me if that is wrong